### PR TITLE
Added createCopies option to disable creating hashed files copies

### PR DESCRIPTION
**Pull request by johny-gog**
I was missing this option because we use server rewrites - so `image.dcf1d324cb50a1f9.png` fetches `image.png`. I just needed to cache bust addresses in our CSS files.

### CODE REVIEW

File: tasks/cachebust.js [hunk 1]
```diff
@@ -59,9 +60,11 @@ module.exports = function() {
             }));
             var newFilename = addFileHash(file, hash, opts.separator);
 
-            grunt.file.copy(absPath, path.resolve(opts.baseDir, newFilename));
+            if(opts.createCopies) {
+                grunt.file.copy(absPath, path.resolve(opts.baseDir, newFilename));
+            }
 
-            if(opts.deleteOriginals) {
+            if(opts.deleteOriginals && opts.createCopies) {
```
Comment (hollandben) [inline tasks/cachebust.js]: I don't see the need for this addition. If someone is going to set `createCopies` to `false` and `deleteOriginals` to `true` then let hell rain down on them

File: README.md [hunk 2]
```diff
@@ -116,11 +117,17 @@ assets: {
...
+Default value: `true`
+
+When set to `false`, `cachebust` will not create hashed copies of the files. Useful if you use server rewrites to serve your files.
+
 #### options.deleteOriginals
 Type: `Boolean`  
 Default value: `false`
 
-When set, `cachebust` will delete the original versions of the files that have been renamed.  For example, `style.css` will be deleted after being copied to `style.dcf1d324cb50a1f9.css`.
+When set, `cachebust` will delete the original versions of the files that have been copied. For example, `style.css` will be deleted after being copied to `style.dcf1d324cb50a1f9.css`. Will not delete original files with `options.createCopies` set to `false`.
```
Comment (hollandben) [inline README.md]: Can we change the word `copied` to `hashed` - that terminology is used in other places in this README. Also, I wouldn't have the addition of that last sentence - see other comment for reasoning

**Comment by hollandben** [comment]
Nice - don't use server rewrites myself but an option that the plugin should cater for. Can you remember to bump the version in the `package.json` and update the changelog please :+1:

File: tasks/cachebust.js (hunk 1 above)
Comment (johny-gog) [inline tasks/cachebust.js]: I guess so... But then which are the `Originals`? :) `Originals` of hashed files, but we don't hash them.
What I would do is create an three-state option with `rename`, `copy` and `original`, but that wouldn't be backward compatible.
Of course we can go without this line. I will make a commit soon.

**Comment by johny-gog** [comment]
Done. :)

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-cache-bust",
   "description": "Bust static assets from the cache using content hashing",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (hollandben) [inline package.json]: By [semver](http://semver.org/) rules it's a new feature that doesn't break backwards compatibility, so should be `1.1.0` - dw, I'll change it when it's merged, just an FYI

**Comment by hollandben** [comment]
`v1.1.0` has been published to npm. Thanks for the contribution :+1:

**Comment by johny-gog** [comment]
Thanks for insta-merge!

**Pull request merged**